app/handler/unsubscribe.py

```diff
@@ -0,0 +1,196 @@
+from email.message import Message
+
+from aiosmtpd.smtp import Envelope
+
+from app.config import URL
+from app.db import Session
+from app.email import headers, status
+from app.email_utils import send_email, render, add_or_replace_header
+from app.log import LOG
+from app.models import Alias, Contact, User, Mailbox
+
+
+class UnsubscribeGenerator:
+    def _generate_header_with_original_behaviour(self, message: Message) -> Message:
```

Comment:
Maybe we can use `@staticmethod` here as this method doesn't depend on `self`? Same for other methods in this file.

Reply:
Yes, it'll probably end up being static but right now it's still in early stages of dev/test :+1: